Formula/sickle.rb

```diff
@@ -17,7 +16,7 @@ def install
       +
       IIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIII0000000000
     EOS
-    output = shell_output("#{bin}/sickle se -f test.fastq -t sanger -o /dev/stdout")
-    assert_match "GTGTC\n", output
+    cmd = "#{bin}/sickle se -f test.fastq -t sanger -o /dev/stdout"
+    assert_match "GTGTC", shell_output(cmd).chomp
```

Comment:
This change no longer tests what I wanted to test, that after running Sickle, the line now ends in `GTGTC`. If Sickle hadn't modified the input at all, this test would still pass. Please revert this change.

Reply:
You could use…
```ruby
assert_equal output.lines[1].chomp, "AGCTTTTCATTCTGACTGCAACGGGCAATATGTCTCTGTGTGGATTAAAAAAAGAGTGTC"
```